### PR TITLE
Lost decorator during log_out

### PR DESCRIPTION
**Pull request by hircumg**
This falls log_out process

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @hircumg! I've created [OSPR-4919](https://openedx.atlassian.net/browse/OSPR-4919) to keep track of it in JIRA, where we prioritize reviews. Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by natabene** [comment]
@hircumg Thank you for the contribution! When you have a chance, could you please send in a signed Contributor Agreement mentioned in the earlier comment? You only need to do this once and we can then review all your future code contributions. Thanks

**Comment by natabene** [comment]
@hircumg Have you had a chance to sign the agreement? Please let me know if you have any questions or need help.

**Comment by hircumg** [comment]
@natabene Yes, I already got a reply about it, as I understood it was from you also. Sorry, that I didn't mention in here.

**Comment by natabene** [comment]
@hircumg You are right, sorry, I missed to update in Github. You are all set. Looks like your PR is ready to be reviewed, correct?

**Comment by hircumg** [comment]
@natabene, I think so.

**Comment by nasthagiri** [comment]
@robrap Do you have any guidance on this change?

File: submission_queue/views.py [hunk 1]
```diff
@@ -34,7 +34,7 @@ def log_in(request):
     else:
         return HttpResponse(compose_reply(False, 'login_required'))
 
-
+@csrf_exempt
```
Comment (robrap) [inline submission_queue/views.py:39]:
```suggestion
# Log out
# --------------------------------------------------
@csrf_exempt
```

**Review by robrap — COMMENTED** [review]
Looks good.  Just want to add a comment for consistency.

**Comment by robrap** [comment]
Also, please squash after accepting the change, and update the commit comment to something like:
```
fix log_out

Adds the `csrf_exempt` decorator to log_out to keep
it from failing.
```
Thank you!

**Comment by natabene** [comment]
@hircumg Have you had a chance to address feedback? Once you do, we can review it again.

**Comment by robrap** [comment]
Hello @hircumg. I'd love to get this merged. Would you be available in the next 1-2 weeks to make the suggested changes? If not, I’d like to take this PR over and merge it myself. Thank you so much for the contribution!

**Comment by hircumg** [comment]
Hello, @robrap. Sure, I will do it in 1-2 weeks.

**Comment by robrap** [comment]
@hircumg: Sounds great.  Thanks!

**Comment by hircumg** [comment]
@robrap 
So,  as I understood squashing possible during merding, like `Squash and merge`.  Hence, I cannot do this because I'm not able to merge.

**Review by robrap — APPROVED** [review]
Great.  Thank you @hircumg for your contribution!

**Comment by openedx-webhooks** [comment]
@hircumg 🎉 Your pull request was merged!

Please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/xqueue&entry.867055334=https://github.com/edx/xqueue/pull/775&entry.1484655318=https://github.com/hircumg&entry.752974735=2020-08-25+07:40&entry.1917517419=2021-01-27+17:20&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**